src/Network/Network/help/Get-AzNetworkManagerSubscriptionConnection.md

```diff
@@ -0,0 +1,132 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:
+schema: 2.0.0
+---
+
+# Get-AzNetworkManagerSubscriptionConnection
+
+## SYNOPSIS
+Gets a network manager subscription connection.
+
+## SYNTAX
+
+### NoExpand (Default)
+```
+Get-AzNetworkManagerSubscriptionConnection [-Name <String>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### Expand
+```
+Get-AzNetworkManagerSubscriptionConnection -Name <String> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzNetworkManagerSubscriptionConnection** cmdlet gets a network manager subscription connection.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzNetworkManagerSubscriptionConnection -Name "testsc"
+
+NetworkManagerId  : /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/PSTestResources/providers/Micros
+                    oft.Network/networkManagers/PSTestNM
+ConnectionState   : Pending
+Description       : SampleDescription
+Type              : Microsoft.Network/networkManagers/networkManagerConnections
+ProvisioningState :
+SystemData        :
+SystemDataText    : null
+Name              : testsc
+Etag              :
+Id                : /subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Network/networkManagerConne
+                    ctions/testsc
+```
+Gets a network manager connection on a subscription.
+
```

Comment:
```suggestion
### Example 2
```

Reply:
Added additional examples to any cmdlets that applied.